src/src/PluginStructs/P105_data_struct.cpp

```diff
@@ -30,15 +30,15 @@ AHTx_Device::AHTx_Device(uint8_t addr, AHTx_device_type type) :
 
 const __FlashStringHelper * AHTx_Device::getDeviceName() const {
   switch (device_type) {
-    case AHT10_DEVICE: return F("AHT10");
-    case AHT20_DEVICE: return F("AHT20");
-    case AHT21_DEVICE: return F("AHT21");
+    case AHTx_device_type::AHT10_DEVICE: return F("AHT10");
+    case AHTx_device_type::AHT20_DEVICE: return F("AHT20");
+    case AHTx_device_type::AHT21_DEVICE: return F("AHT21");
     default: return F("AHTx");
```

Comment:
You could leave out the "default" part and just return this outside the switch scope.
Then the compiler will warn you when you missed one (if you add one later)
Using `default:` removes this kind of checks as the default option makes all possible values being handled.